portfolio/src/main/webapp/script.js

```diff
@@ -76,4 +76,14 @@ function addMessageToDom(message) {
   console.log('Adding message to dom: ' + message);
 
   const messageContainer = document.getElementById('message-container');
-  messageContainer.innerText = message;
\ No newline at end of file
+  messageContainer.innerText = message;
+}
+
+  /**
+ * Fetches messages from the servers and adds them to the DOM.
+ */
+function getMessagess() {
```

Comment:
It looks like this isn't called anywhere.  Is that intentional?

Reply:
Also looks like a typo: Messages*s*

Reply:
Thank you.